cli/core/validator.go

```diff
@@ -109,11 +109,16 @@ func GenerateValidatorProof(ctx context.Context, eigenpodAddress string, eth *et
 		return nil, 0, fmt.Errorf("failed to fetch beacon state: %w", err)
 	}
 
-	proofs, err := GenerateValidatorProofAtState(eigenpodAddress, beaconState, eth, chainId, header, latestBlock.Time())
+	proofExecutor, err := eigenpodproofs.NewEigenPodProofs(chainId.Uint64(), 300 /* oracleStateCacheExpirySeconds - 5min */)
```

Comment:
this doesn't help speed up the CLI, which only makes 1 call to proveValidators, it only speed up when we make successive calls to proveValidators for the same beacon state

Reply:
this `core` module is used in the serfer. it's just sitting in the wrong directory rn :)